nixos/lib/test-driver/test_driver/__init__.py

```diff
@@ -105,8 +105,14 @@ def main() -> None:
         args.output_directory.resolve(),
         args.keep_vm_state,
     ) as driver:
+        history_dir = os.getcwd()
+        history_path = os.path.join(history_dir, ".nixos-test-history")
```

Comment:
can you move these vars into the if-clause? it seems like that's the smallest useful scope.

Reply:
done